locales/en-US/app.ftl

```diff
@@ -51,6 +51,17 @@ errorMessage = Looks like we broke something. <br> Maybe try again later.
 
 notFoundHeader = Four Oh Four!
 
+emailOptInDialogTitle = Welcome to Test Pilot!
+emailOptInMessage = Find out about new experiments and see test results for experiments you've tried.
+emailValidationError = Please use a valid email address!
+# LOCALIZATION NOTE: The ':)' characters in emailOptInPlaceholder are a smiley face emoticon.
```

Comment:
Was about to ask if this did the right thing, but [apparently so](http://l20n.org/learn/comments)!
> If a comment is located right above section or entity, it belongs to it and localization tools will present it in its context.